bus-mapping/src/precompile.rs

```diff
@@ -3,6 +3,20 @@
 use eth_types::Address;
 use revm_precompile::{Precompile, Precompiles};
 
+/// addresses and configs for scroll's l2 precompile
+pub mod l2_address {
+    use super::*;
+    use eth_types::U256;
+    use once_cell::sync::Lazy;
+    use std::str::FromStr;
+
+    /// address of L2MessageQueue predeploy
+    pub static MESSAGE_QUEUE: Lazy<Address> =
+        Lazy::new(|| Address::from_str("0x5300000000000000000000000000000000000000").unwrap());
+    /// the slot of withdraw root in L2MessageQueue
+    pub static WITHDRAW_TRIE_ROOT_SLOT: Lazy<U256> = Lazy::new(U256::zero);
+}
```

Comment:
no i don't think MESSAGE_QUEUE can be called "precompile".   They can be named "l2_predeployed_contracts"

Reply:
Now move the constants into `bus_mapping::l2_predeployed::message_queue`